### ISSUE TITLE
Error loading plugins: gitbook-plugin-json

### ISSUE THREAD

**Issue by kokoye2007** [pm5/gitbook-plugin-json#4]
```
info: loading book configuration....OK 
info: load plugin gitbook-plugin-json ....ERROR
info: load plugin gitbook-plugin-highlight ....OK 
info: load plugin gitbook-plugin-search ....OK 
info: load plugin gitbook-plugin-sharing ....OK 
info: load plugin gitbook-plugin-fontsettings ....OK 

Error: Error loading plugins: gitbook-plugin-json. Run 'gitbook install' to install plugins from NPM.
```

but gitbook insall is done

```
info: 1 plugins to install 
info: No version specified, resolve plugin json 
info: install plugin json from npm (gitbook-plugin-json) with version 0.0.8 
info: >> plugin json installed with success 

Done, without error
```

**Comment by kokoye2007** [comment]
now ok

```
npm i javascript-natural-sort
```

**Comment by pm5** [comment]
Resolved in version 1.0.0.
